roster2/bundle/src/bundle/card_game_ca.properties

```
@@ -2,6 +2,7 @@ check_hit_info = Correcte! L\u0027estudiant era {0}!
 check_miss_info = Ups, aix\u00F2 est\u00E0 malament! El nom d\u0027aquest estudiant \u00E9s {0}.
 check = Comprobar
 close = Tancar
+continue = Continua
```

```suggestion
continue = Continuar
```
If the previous cases use the infinitive and the Spanish does too I think it's better like this

---

I copied this translation from another bundle, but I'll commit your change. Thanks